### ISSUE TITLE
verifier problem

### ISSUE THREAD

**Issue by qq1739920004** [hubofvalley/Testnet-Guides#2]
I ran the verifier following your tutorial, And I can give it to my depositor normally
How do I see which address is the verifier I'm running? Because I imported several wallets, several wallets are created
My priv_validator_state.json content is ![image](https://github.com/user-attachments/assets/85d2619c-50f6-45ae-b905-8692dd284c3e)
My priv_validator_state.json content is 
![image](https://github.com/user-attachments/assets/61da1b19-6b72-40b6-866f-c3f1133dd96f)

**Comment by hubofvalley** [comment]
your validator is not in the active list.`` priv_validator_state.json`` is the signture file that u've signed block, and validator who can signs block is validator who are included in the active list. that's the base disclaimer. now about your question, u can check your validator info in the explorer https://explorer.grandvalleys.com/0g-chain%20testnet

**Comment by qq1739920004** [comment]
> 你的验证者不在活跃列表中。` priv_validator_state.json`签名文件是你签名的块，可以签名块的验证者是包含在活跃列表中的验证者。这是基本免责声明。现在关于你的问题，你可以在浏览器中查看你的验证者信息https://explorer.grandvalleys.com/0g-chain%20testnet
I didn't find me among the inactive verifiers either
0g1dl0n2gx267ew75rmxrgk97y0837mg6ltnz7hge

Can I get my validator address from this priv_validator_state.json content?

**Comment by qq1739920004** [comment]
> your validator is not in the active list.` priv_validator_state.json` is the signture file that u've signed block, and validator who can signs block is validator who are included in the active list. that's the base disclaimer. now about your question, u can check your validator info in the explorer https://explorer.grandvalleys.com/0g-chain%20testnet

I didn't find me among the inactive verifiers either
0g1dl0n2gx267ew75rmxrgk97y0837mg6ltnz7hge

Can I get my validator address from this priv_validator_state.json content?

**Comment by qq1739920004** [comment]
Does the contents of the validator priv_validator_key.json change every time you create it? I think I created one out of three wallets？

**Comment by qq1739920004** [comment]
> your validator is not in the active list.` priv_validator_state.json` is the signture file that u've signed block, and validator who can signs block is validator who are included in the active list. that's the base disclaimer. now about your question, u can check your validator info in the explorer https://explorer.grandvalleys.com/0g-chain%20testnet

I didn't find me among the inactive verifiers either
0gvaloper1dl0n2gx267ew75rmxrgk97y0837mg6ltsg0tay
![image](https://github.com/user-attachments/assets/85fb1e71-7739-4375-a028-ad524ec717fd)


Maybe what's in my priv_validator_state. json file isn't this validator, because I've created validators in multiple wallets
Can I get my validator address from this priv_validator_state.json content?

**Comment by hubofvalley** [comment]
u cna search your valoper address in the explorer

**Comment by hubofvalley** [comment]
btw i've found your address, check it out!
https://explorer.grandvalleys.com/0g-chain%20testnet/account/0g1dl0n2gx267ew75rmxrgk97y0837mg6ltnz7hge

**Comment by qq1739920004** [comment]
> 顺便说一句，我找到了你的地址，看看吧！ https://explorer.grandvalleys.com/0g-chain%20testnet/account/0g1dl0n2gx267ew75rmxrgk97y0837mg6ltnz7hge


Thank you. I’m planning to create a new wallet and set up a new validator. I just deployed a node on a new machine and completed the sync, but I haven’t started creating the validator for my wallet yet. I noticed that the priv_validator_state.json file already contains data, even though I haven't created the validator. Does this mean that the pub_key and pri_key in that file are the keys that will be used for the validator I'm about to create with this new wallet?

**Comment by qq1739920004** [comment]
> btw i've found your address, check it out! https://explorer.grandvalleys.com/0g-chain%20testnet/account/0g1dl0n2gx267ew75rmxrgk97y0837mg6ltnz7hge




Thank you. I’m planning to create a new wallet and set up a new validator. I just deployed a node on a new machine and completed the sync, but I haven’t started creating the validator for my wallet yet. I noticed that the priv_validator_state.json file already contains data, even though I haven't created the validator. Does this mean that the pub_key and pri_key in that file are the keys that will be used for the validator I'm about to create with this new wallet?
